### PR TITLE
PMM-5064 Alert manager settings.

### PR DESCRIPTION
**Pull request by percona-csalguero**
Added alert manager fields to settings

https://github.com/Percona-Lab/pmm-submodules/pull/645

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

File: api/serverpb/server.proto [hunk 1]
```diff
@@ -146,6 +148,10 @@ message ChangeSettingsRequest {
   google.protobuf.Duration data_retention = 4;
   string ssh_key = 5;
   repeated string aws_partitions = 6;
+  string alert_manager_address = 7;
+  bool clear_alert_manager_address = 8;
```
Comment (AlekSi) [inline api/serverpb/server.proto]: Let's use `remove_` prefix as in Change Agent API.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

File: api/serverpb/server.proto [hunk 2]
```diff
@@ -130,6 +130,9 @@ message Settings {
   google.protobuf.Duration data_retention = 4;
   string ssh_key = 5;
   repeated string aws_partitions = 6;
+  // Prometheus AlertManager URL (e.g., https://1.2.3.4/path).
+  string alert_manager_url = 7;
```
Comment (AlekSi) [inline api/serverpb/server.proto]: We need a full URL there, with schema and path.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-477/

**Pull request merged**